src/specific/s_input.c

```diff
@@ -56,19 +67,19 @@ static INPUT_SCANCODE m_Layout[INPUT_LAYOUT_NUMBER_OF][INPUT_ROLE_NUMBER_OF] = {
 
     // custom user controls
     {
-        SDL_SCANCODE_KP_8,       // INPUT_ROLE_UP
-        SDL_SCANCODE_KP_2,       // INPUT_ROLE_DOWN
-        SDL_SCANCODE_KP_4,       // INPUT_ROLE_LEFT
-        SDL_SCANCODE_KP_6,       // INPUT_ROLE_RIGHT
-        SDL_SCANCODE_KP_7,       // INPUT_ROLE_STEP_L
-        SDL_SCANCODE_KP_9,       // INPUT_ROLE_STEP_R
-        SDL_SCANCODE_KP_1,       // INPUT_ROLE_SLOW
-        SDL_SCANCODE_KP_PLUS,    // INPUT_ROLE_JUMP
-        SDL_SCANCODE_KP_ENTER,   // INPUT_ROLE_ACTION
-        SDL_SCANCODE_KP_3,       // INPUT_ROLE_DRAW
+        SDL_SCANCODE_UP,         // INPUT_ROLE_UP
+        SDL_SCANCODE_DOWN,       // INPUT_ROLE_DOWN
+        SDL_SCANCODE_LEFT,       // INPUT_ROLE_LEFT
+        SDL_SCANCODE_RIGHT,      // INPUT_ROLE_RIGHT
+        SDL_SCANCODE_DELETE,     // INPUT_ROLE_STEP_L
+        SDL_SCANCODE_PAGEDOWN,   // INPUT_ROLE_STEP_R
+        SDL_SCANCODE_RSHIFT,     // INPUT_ROLE_SLOW
+        SDL_SCANCODE_RALT,       // INPUT_ROLE_JUMP
+        SDL_SCANCODE_RCTRL,      // INPUT_ROLE_ACTION
+        SDL_SCANCODE_SPACE,      // INPUT_ROLE_DRAW
```

Comment:
Why do you change the second layout to match the default layout?
I remember these values were taken from TombATI.

Reply:
I changed every control scheme to start as the default controls because we don't have separate menu controls. If these were left, when the user scrolls to this control scheme in the menu, the arrow keys would no longer work. The up/down/left/right would now be mapped to the number pad which seemed intuitive to me.